ADMVEDU95/ADMVEDU95/ViewControllers/HomeViewController/HomeViewController.swift

```diff
@@ -9,31 +9,23 @@ import UIKit
 
 class HomeViewController: UIViewController {
     struct Constants {
-        static let searchCellIdentifier = "SearchResponceTableViewCell"
+        static let searchCellIdentifier = "SearchTableViewCell"
```

Comment:
we can avoid additional string constants and use something like SearchTableViewCell.reuseIdentifier which returns the "SearchTableViewCell" string. Please think his you'd implement it